development/cpp/configuring_an_ide/android_studio.rst

```diff
@@ -4,15 +4,14 @@ Android Studio
 ==============
 
 `Android Studio <https://developer.android.com/studio>`_ is a free
-`JetBrains <https://www.jetbrains.com/>`_ IDE for Android development.
+IDE for Android development based on `Intellij <https://www.jetbrains.com/>`__.
```

Comment:
Well, if we still link to the JetBrains website, it should also be okay to mention them by name. Android Studio *is* developer by both JetBrains and Google, so this statement wasn't wrong.

And if we were to mention the parent product, it's *IntelliJ IDEA*, not *Intellij*.